homeassistant/components/switchbot/strings.json

```
@@ -58,5 +58,56 @@
         }
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "calibration": {
+        "name": "Calibration"
+      },
+      "motion": {
+        "name": "[%key:component::binary_sensor::entity_component::motion::name%]"
+      },
+      "door_open": {
+        "name": "[%key:component::binary_sensor::entity_component::door::name%]"
+      },
+      "door_timeout": {
+        "name": "Timeout"
+      },
+      "light": {
+        "name": "[%key:component::binary_sensor::entity_component::light::name%]"
+      },
+      "door_unclosed_alarm": {
+        "name": "Unclosed alarm"
+      },
+      "door_unlocked_alarm": {
+        "name": "Unlocked alarm"
+      },
+      "door_auto_lock_paused": {
+        "name": "Auto-lock paused"
+      }
+    },
+    "sensor": {
+      "bluetooth_signal": {
+        "name": "Bluetooth signal"
+      },
+      "wifi_signal": {
+        "name": "Wi-Fi signal strength"
```

Should I do the same as Bluetooth and remove "strength"?

```suggestion
        "name": "Wi-Fi signal"
```

---

Yes